versioned_docs/version-10.x/eloquent-mutators.md

```
@@ -0,0 +1,874 @@
+# Eloquent: 접근자 & 캐스팅 (Eloquent: Mutators & Casting)
+
+- [소개](#introduction)
+- [접근자와 변이자](#accessors-and-mutators)
+    - [접근자 정의하기](#defining-an-accessor)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)